get-started/get-started.adoc

```diff
@@ -9,6 +9,13 @@
 Taking the first steps with Neo4j?
 Read our introductory tutorials and developer guides that cover all areas of your development work.
 
+These pages were written to guide you through much of your initial learning for Neo4j and give you resources to continue your education.
```

Comment:
edit: "These **introductory tutorials and developer guides** were written to..."

Reply:
That's better.